### ISSUE TITLE
Selecting scatter changeds content

### ISSUE THREAD

**Issue by Favkis** [HungryProton/scatter#78]
https://user-images.githubusercontent.com/46406204/182555792-dc79f22f-f3b6-4e60-a2a2-81cce66d18b4.mp4

Selecting scatter item regenerates contents or something. I dont know what's happening but it looks like it's stuck between two states, and they cycle depending if scatter is selected.

here's that place in game 
![image](https://user-images.githubusercontent.com/46406204/182557278-264998ae-ce77-4c07-a701-c7371287958b.png)
and in editor 
![image](https://user-images.githubusercontent.com/46406204/182557364-56f4a06e-46ad-44dc-b98b-17c33cfb4844.png)

**Comment by HungryProton** [comment]
Yes, selecting a scatter node recalculates its content positions, but unless you have a script changing the seeds, the results should be exactly the same. 

The sync issue might be fixed by disabling `Force update when loaded` but that's not the root cause of it.

I can't reproduce the issue on my end, so I suspect something is happening with the scripts attached to the rocks.

**Comment by Favkis** [comment]
> Yes, selecting a scatter node recalculates its content positions, but unless you have a script changing the seeds, the results should be exactly the same.
> 
> The sync issue might be fixed by disabling `Force update when loaded` but that's not the root cause of it.
> 
> I can't reproduce the issue on my end, so I suspect something is happening with the scripts attached to the rocks.

script only changes MeshInstanced inside of it and CollisionShape.disabled=true depending on distance to player, it doesnt touch anything else

**Comment by Favkis** [comment]
Force update doesn't help with it btw.
What I did is I rotated scatter node 90 Degrees by X because when I do curves they cant go down for some reason (whole line is at minimal distance of lowest point) so to make vertical strips I had to rotate it by X

**Comment by HungryProton** [comment]
Yes, that's by design, but it should work fine even if the node is rotated. 

If you can share a minimal reproduction project that would help a lot, I have no idea what could be causing these issues.
